### PR TITLE
Add door speed changes to remotes

### PR DESCRIPTION
**Pull request by FlattestGuitar**
:cl: Flatty
add: Door remotes can now be used to change the speed of airlocks.
:cl:

### CODE REVIEW

File: code/game/objects/items/control_wand.dm [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #define WAND_OPEN "Open Door"
 #define WAND_BOLT "Toggle Bolts"
 #define WAND_EMERGENCY "Toggle Emergency Access"
+#define WAND_SPEED "Change Door Speed"
```
Comment (farie82) [inline code/game/objects/items/control_wand.dm:4]: "Change Closing Speed"?

File: code/game/objects/items/control_wand.dm [hunk 2]
```diff
@@ -64,10 +68,13 @@
...
 				if(D.emergency)
-					D.emergency = 0
+					D.emergency = FALSE
 				else
-					D.emergency = 1
+					D.emergency = TRUE
 				D.update_icon()
+			if(WAND_SPEED)
+				D.normalspeed = !D.normalspeed
+				to_chat(user, "Now in [D.normalspeed ? "normal" : "fast"] mode.")
```
Comment (farie82) [inline code/game/objects/items/control_wand.dm:77]: Needs spans here. notice will do as class.
Text also needs some more flavor. "Now in X mode" is a bit flavorless if you know what I mean

**Review by farie82 — CHANGES_REQUESTED** [review]

**Review by farie82 — APPROVED** [review]

**Review by variableundefined — APPROVED** [review]
Approved for design

**Review by Kyep — APPROVED** [review]

**Review by AffectedArc07 — APPROVED** [review]
Stop bugging me for hacktoberfest

**Pull request merged**